src/gui/src/displayControls.cpp

```diff
@@ -509,14 +509,35 @@ void DisplayControls::setDb(odb::dbDatabase* db)
   emit changed();
 }
 
-template <typename T>
-QStandardItem* DisplayControls::makeItem(
+QStandardItem* DisplayControls::makeParentItem(
+    ModelRow& row,
+    const QString& text,
+    QStandardItemModel* parent,
+    Qt::CheckState checked,
+    const CallbackFunction& visibility_action,
+    const CallbackFunction& select_action)
```

Comment:
I don't think these are needed as the actions are always the same at each caller and just depend on other args.  They can be created in this method.